src/pages/trade-types/options/index.tsx

```diff
@@ -21,12 +21,12 @@ const meta_attributes = {
 }
 
 const Options = () => {
-    const { is_row } = useCountryRule()
+    const { is_row } = React.useContext(RegionContext)
     const [is_loaded, setLoaded] = useState(false)
 
     useEffect(() => {
         setLoaded(true)
```

Comment:
@mohammad-rework What’s the point of doing this? 🤔 I think we can remove the `useEffect` and just set the initial value of `is_loaded` to `true` 🤔